src/backend.py

```diff
@@ -155,3 +156,11 @@ def handle_status_code(status_code):
     async def get_wow_character_achievements(self,  realm, character_name):
         url = f"https://{self._authentication_client.region}.api.blizzard.com/wow/character/{realm.lower()}/{character_name}?fields=achievements"
         return await self.do_request("GET", url)
+
+    async def get_ow_player_data(self):
+        url = 'https://owapi.io/profile/pc/' + self._authentication_client.region + '/' + self._authentication_client.user_details['battletag'].replace('#', '-')
```

Comment:
Let's use f-strings for consistency. 

And didn't you forget "complete" in this url? Like
"https://ow-api.com/v1/stats/pc/eu/saitho%232703/complete" instead of "https://ow-api.com/v1/stats/pc/eu/saitho%232703"
```suggestion
        url = f"https://owapi.io/profile/pc/{self._authentication_client.region}/{self._authentication_client.user_details['battletag'].replace('#', '-')}/complete"
```

Reply:
this suggestion need to be tested

Reply:
> And didn't you forget "complete" in this url?

No, @saitho used https://owapi.io ([docs](https://owapi.io/docs/#api-Profile-GetProfile)) now, not https://ow-api.com - it is a different (third party) api.